src/config/tokens.ts

```diff
@@ -536,18 +568,33 @@ export function buildTokenCache(
   tokens: TokenConfig[],
   wrappedTokens: WrappedTokenAddresses,
   cacheKey: string,
+  customConfig: WormholeConnectConfig = {},
 ): TokenCache {
   const cache = TokenCache.load(cacheKey);
-
+  const network = capitalize(
+    customConfig.network ||
+      import.meta.env.REACT_APP_CONNECT_ENV?.toLowerCase() ||
+      'Mainnet',
+  ) as Network;
   for (const { tokenId, symbol, name, icon, decimals } of tokens) {
-    const token = new Token(
-      tokenId.chain,
-      tokenId.address.toString(),
+    const wrappedToken = getWrappedNativeToken(network, tokenId.chain);
+    const isWrappedToken =
+      wrappedToken &&
+      tokenId.address.toLowerCase() === wrappedToken.toLowerCase();
+    const token = new Token({
+      chain: tokenId.chain,
+      address: tokenId.address.toString(),
       decimals,
       symbol,
       name,
       icon,
-    );
+      tokenBridgeOriginalTokenId: isWrappedToken
```

Comment:
A couple of things:
1. We should not set the `tokenBridgeOriginalTokenId` property unless this is actually a token bridge wrapped token. The `getWrappedNativeToken` function above returns the wrapped version of the native/gas token, which is not a token bridge wrapped token. To determine if a token is a token bridge wrapped token, we have to call:

```typescript
const tb = await chain.getTokenBridge(); // where `chain` is the token's chain
if (await tb.isWrappedAsset(tokenId.address)) {
  tokenBridgeOriginalTokenId = await tb.getOriginalAsset(tokenId.address);
```
But since we can't and don't want to call this for every token when building the cache, we have to do something different.

2. What did adding this fix? Maybe we can work backwards from there to find a fix.